docker/app/Dockerfile

```diff
@@ -0,0 +1,90 @@
+# syntax=docker/dockerfile:1
+ARG RUBY_VERSION=3.4.3
+FROM ruby:${RUBY_VERSION}-slim-bullseye

```

Comment:
Makes sense to use `*-bullseye` image here since we don't have the latest debian release for Ruby 3.4.3. Could you add a comment here we can use the latest OS when we upgrade to Ruby 3.4.7?